package.json

```diff
@@ -4,8 +4,8 @@
   "main": "src/web/pages",
   "private": true,
   "scripts": {
-    "dev": "cd src/web && next dev",
-    "build": "next build",
+    "dev": "next dev src/web",
```

Comment:
The script does not seem to run locally anymore

Reply:
For some reason, it works for me. Can anyone else confirm this?
```
 npm run dev

> boardly@0.1.0 dev
> next dev src/web

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
warn  - React 17.0.1 or newer will be required to leverage all of the upcoming features in Next.js 11. Read more: https://err.sh/next.js/react-version
event - compiled successfully
```

Reply:
The script still doesnt work?

Reply:
Works now